<launch>
 <param name="CONFIG" value="$(find dialog_system)/corpus/dialog_iai.corpus"/>

 <param name="RPCSERVERPORT" value="8000"/>

 <param name="INDEX" value="1"/>
 <param name="NBTHREADS" value="4"/>
 <param name="BEAMSIZE" value="3"/>
 <param name="HOST" value="192.168.178.38" />
 <param name="PORT" value="7000"/>
 <param name="RPCPORT" value="8000"/>
 <param name="DATAPATH" value="data"/>
 <param name="ASRPATH" value="pocketsphinx_continuous"/>
 <param name="ASRCWD" value="$(find dialog_system)"/>
 <param name="TRESHOLD" value="-10000"/>
 <param name="HMM" value="model/en-us-adapt"/>
 <param name="MLLR" value="model/mllr_matrix"/>

 <param name="ORDER" value="0"/>

 <include file="$(find knowrob_pepper_dialog)/launch/pepper_dialog.launch" />

 <node name="rpc_server" pkg="dialog_system" type="rpc_server.py" output="screen">
  </node>

 <node name="dialogManager" pkg="dialog_system" type="dialogManager.py" output="screen">
  </node>

 <node name="sphinx_asr" pkg="dialog_system" type="sphinx_asr.py" output="screen">
  </node>

</launch>
